src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -1324,21 +1331,17 @@ public FixedPlattCalibratorTrainer(IHostEnvironment env, Arguments args)
             _offset = args.Offset;
         }
 
-        public bool NeedsTraining => false;
+        bool ICalibratorTrainer.NeedsTraining => false;
 
-        public bool ProcessTrainingExample(Float output, bool labelIs1, Float weight)
-            => false;
+        bool ICalibratorTrainer.ProcessTrainingExample(float output, bool labelIs1, float weight) => false;
 
-        public ICalibrator FinishTraining(IChannel ch)
-        {
-            return new PlattCalibrator(_host, _slope, _offset);
-        }
+        ICalibrator ICalibratorTrainer.FinishTraining(IChannel ch) => new PlattCalibrator(_host, _slope, _offset);
     }
 
     public sealed class PlattCalibrator : ICalibrator, IParameterMixer, ICanSaveModel, ISingleCanSavePfa, ISingleCanSaveOnnx
```

Comment:
<summary> Platt calibrator. Calculate probability following way P(x) = 1 / (1 + exp(-slope * x + offset) </> (make ref for slope and offset).